android-core/src/main/kotlin/com.mparticle/NetworkUtilities.kt

```diff
@@ -0,0 +1,26 @@
+package com.mparticle
+
+object NetworkUtilities {
+    fun getUrlWithPrefix(
+        url: String?,
+        podPrefix: String,
+        enablePodRedirection: Boolean
+    ): String? {
+        return url?.let {
+            val newUrl = if (enablePodRedirection) {
+                "$it.$podPrefix"
+            } else {
+                "$it"
+            }
+            "$newUrl.mparticle.com"
+        }
+    }
+
+    fun getPodPrefix(apiKey: String): String? {
```

Comment:
let's move this to a test class since it is not being consumed in the core sdk at all